slave-services/src/main/java/org/iqu/slaveservices/entities/News.java

```diff
@@ -8,30 +8,36 @@
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
 
```

Comment:
Probably in the same package: **org.iqu.slaveservices.rest.consumer**

Also make it not public but default package ... Change the comment to something more meaningful like: **Entity used to consume news from rest services.**